### PR TITLE
rodrigovcesar/c02-network07

### PR DESCRIPTION
**Pull request by rodrigovcesar**
# Description

Exercise (e.g. C01-E01): c02-network07

Student Name: Rodrigo Vasconcellos Cesar

> Answers for the exercise c02-network07

## Type of change

- [X] Exercise Submission (one exercise per PR)
  - [X] This PR is just for **one** exercise of a class
  - [X] PR name follows the pattern `<github-username>/<exercise-number>`
  - [X] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/02class/exercises/c02-network07/rodrigovcesar/README.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    ```
+    
+  - Can you delete the destination route to your VPC network? Why?
+    ```
+    - No, because it's a local route, otherwise it would cause disruption on the communication between the subnets and the VPC.
+    ```
+    
+  - What happens if you do not associate your route table with any subnets?
+    ```
+    Nothing. A route table without any associated subnet is not useful.
```
Comment (LuizFDS) [inline classes/02class/exercises/c02-network07/rodrigovcesar/README.md:186]: So which route table will be used instead?

Comment (rodrigovcesar) [reply classes/02class/exercises/c02-network07/rodrigovcesar/README.md]: Hi Luiz,

I made a change on the answer and tried to explain more. Thank you for the feedback.

File: classes/02class/exercises/c02-network07/rodrigovcesar/README.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+}
+
+aws ec2 create-tags --resources rtb-013716806215ecf49 --tags Key=Name,Value=devopsacademy-rt-private
+
+
+```
+
+- Commands for associating the route tables with subnets:
+```
+aws ec2 describe-subnets --filters "Name=tag:Name,Values=public-a,public-b,public-c" --query 'Subnets[*].SubnetId'
```
Comment (roperto) [inline classes/02class/exercises/c02-network07/rodrigovcesar/README.md:58]: nice listing the subnets

File: classes/02class/exercises/c02-network07/rodrigovcesar/README.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+    
+  - What happens if you do not associate your route table with any subnets?
+    ```
+    All VPCs have one main table route (RT) by default, controlling the routing for all subnets that are not explicitly associated with any RT.  
+    
+    Every subnet, when created and not explicitly assigned with any RT, is automatically associated with the main route table from the VPC - an implicity association.   
+    
+    A subnet can only be associated with one RT at a time, but you can associate multiple subnets with the same subnet RT.
+    
+    In this case, from what I understand,  creating a custom RT without any association will not produce any effect at all if you do not have an explicit association.
```
Comment (roperto) [inline classes/02class/exercises/c02-network07/rodrigovcesar/README.md:192]: Correct, the route will stay there unused (unless you make it as default route, which could get the unassociated subnets)

**Pull request merged**